src/MultiTypeSymEntry.chpl

```diff
@@ -22,6 +26,10 @@ module MultiTypeSymEntry
         return gse.toSymEntry(etype);
     }
 
+    inline proc toSegStringSymEntry(gse: borrowed GenSymEntry) {
+        return try! gse: borrowed SegStringSymEntry(string);
```

Comment:
Although we had a lot of `try!` in the initial codebase, lately we've been trying to avoid it because it halts the server. Is it possible to get rid of the `try!` and make this a throwing function, or is this a special case where that isn't allowed? I forget the rules...

Reply:
I talked to @mhmerrill about that too.  This follows the current `toSymEntry(type etype)` logic which is a `try!` because I didn't want to deviate from the current behavior (line 52 on master branch).